modules/apps/headless/headless-admin-user/headless-admin-user-impl/src/main/java/com/liferay/headless/admin/user/internal/resource/v1_0/AccountResourceImpl.java

```diff
@@ -670,6 +639,35 @@ null, _getType(account), _getStatus(account),
 		return _toAccount(accountEntry);
 	}
 
+	private void _addAddresses(Long accountId, Account account)
+		throws Exception {
+
+		PostalAddress[] postalAddresses = account.getPostalAddresses();
+
+		if ((postalAddresses != null) && (postalAddresses.length > 0)) {
```

Comment:
if (postalAddresses == null or postalAddresses.length == 0) {
  return Collections.emptyList();
} so we can unindent the following code